specification/translation/data-plane/Azure.AI.DocumentTranslation/stable/2024-05-01/openapi.json

```diff
@@ -0,0 +1,1463 @@
+            "required": false,
+            "type": "integer",
+            "format": "int32",
+            "maximum": 2147483647
+          },
+          {
+            "name": "skip",
+            "in": "query",
+            "description": "skip indicates the number of records to skip from the list of records held by\nthe server based on the sorting method specified.  By default, we sort by\ndescending start time.\n            \nClients MAY use top and skip query\nparameters to specify a number of results to return and an offset into the\ncollection.\nWhen both top and skip are given by a client, the server SHOULD\nfirst apply skip and then top on the collection.\n            \nNote: If the\nserver can't honor top and/or skip, the server MUST return an error to the\nclient informing about it instead of just ignoring the query options.",
+            "required": false,
+            "type": "integer",
+            "format": "int32",
+            "maximum": 2147483647
+          },
+          {
+            "name": "maxpagesize",
+            "in": "query",
+            "description": "maxpagesize is the maximum items returned in a page.  If more items are\nrequested via top (or top is not specified and there are more items to be\nreturned), @nextLink will contain the link to the next page.\n           \n\nClients MAY request server-driven paging with a specific page size by\nspecifying a maxpagesize preference. The server SHOULD honor this preference\nif the specified page size is smaller than the server's default page size.",
+            "required": false,
+            "type": "integer",
+            "format": "int32",
+            "default": 50,
+            "minimum": 1,
+            "maximum": 100
+          },
+          {
+            "name": "ids",
+            "in": "query",
+            "description": "Ids to use in filtering",
+            "required": false,
+            "type": "array",
+            "items": {
+              "type": "string",
+              "format": "uuid"
+            },
+            "collectionFormat": "csv"
+          },
+          {
+            "name": "statuses",
+            "in": "query",
+            "description": "Statuses to use in filtering",
+            "required": false,
+            "type": "array",
+            "items": {
+              "type": "string"
+            },
+            "collectionFormat": "csv"
+          },
+          {
+            "name": "createdDateTimeUtcStart",
+            "in": "query",
+            "description": "the start datetime to get items after",
+            "required": false,
+            "type": "string",
+            "format": "date-time"
+          },
+          {
+            "name": "createdDateTimeUtcEnd",
+            "in": "query",
+            "description": "the end datetime to get items before",
+            "required": false,
+            "type": "string",
+            "format": "date-time"
+          },
+          {
+            "name": "orderBy",
```

Comment:
The orderby parameter should be named "orderby" (all lowercase).

Reply:
Updated.